src/main/cpp/auton/primitives/DrivePath.cpp

```diff
@@ -119,12 +163,129 @@ bool DrivePath::lRobotStopped(frc::Pose2d lCurPos, frc::Pose2d lPrevPos)
   }
   return lresultStopped;
 }
-/*  NOTES Users can create their own constraint by implementing the TrajectoryConstraint interface.
-SwerveDriveKinematicsConstraint: Limits the velocity of the robot around turns
-such that no wheel of a swerve-drive robot goes over a specified maximum velocity.
+std::string DrivePath::lGetGSPathFromVisionTbl()
```

Comment:
I would have put this method into auton select or a separate class that auto select calls